src/Calypso-SystemTools-FullBrowser/ClyFullBrowserMorph.class.st

```diff
@@ -129,6 +129,20 @@ ClyFullBrowserMorph class >> registerToolsOn: registry [
 	registry register: self as: #browser
 ]
 
+{ #category : #settings }
+ClyFullBrowserMorph class >> settingsDragAndDropDelayOn: aBuilder [
+	<systemsettings>
+	(aBuilder setting: #dragAndDropDelay)
```

Comment:
And here it would be "aBuilder setting: #dragAndDropDelaySeconds"
I would also define this method on ClyQueryViewMorph directly because it's a configuration of this widget and not the full browser.